pandas/core/frame.py

```diff
@@ -6483,76 +6485,72 @@ def join(self, other, on=None, how='left', lsuffix='', rsuffix='',
         Support for specifying index levels as the `on` parameter was added
         in version 0.23.0
```

Comment:
Do you mind capitalizing the first letter of each sentence, and ending them in period in the Notes. For the First paragraph, if you can add backticks around the parameters `on`, `lsuffix`... that would be great too. So: "Options \`on\`, \`lsuffix\` and \`rsuffix\` are not supported [...]"